docs/content/release/0.25.0-notes.rst

```diff
@@ -0,0 +1,367 @@
+.. _0.25.0_notes:
+
+============================
+SimPEG v0.25.0 Release Notes
+============================
+
+October 21st, 2025
+
+.. contents:: Highlights
+    :depth: 3
+
+Updates
+=======
+
+New features
+------------
+
+New differential simulation for magnetic fields
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This release ships with a new and improved version of the magnetic differential
+simulation
+:class:`~simpeg.potential_fields.magnetics.Simulation3DDifferential` by
+`@johnweis0480 <https://github.com/johnweis0480>`__.
+This simulation computes the magnetic field on every cell of the mesh by
+numerically solving the magnetostatic PDE. It accounts for
+self-demagnetization effects, model both induced and remanent magnetizations,
+and is faster and less memory intensive than the integral simulation for large
+problems.
+
+See https://github.com/simpeg/simpeg/pull/1682 for more details.
+
+Add utility function to shift electrodes to discrete topography
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A new `shift_to_discrete_topography` function shifts locations relative
+to discrete surface topography. When performing MT surveys, we measure
+electric fields at the Earth's surface. Similar to DC/IP, the original measurement
+locations of the electric fields can end up in air cells when we discretize surface
+topography. This function allows the user to shift locations
+relative to discrete topography on Tensor and Tree meshes. For Airborne NSEM,
+they also allow the user to preserve the original flight heights.
+
+See https://github.com/simpeg/simpeg/pull/1683 for more details.
+
+Use a closed-loop wire as a source in TDEM
```

Comment:
```suggestion
Calculate B/H fields with a step-off waveform closed-loop wire source in TDEM
```
(we could still use a closed-loop wire source before, but only to get db/dt or E field data)

Reply:
Thanks Lindsey. I'm commiting this.